packages/fluxible-plugin-devtools/src/lib/devtools-plugin.js

```diff
@@ -144,6 +144,10 @@ export default function devToolsPlugin() {
                         return action(ctx, payload, cb);
                     }
 
+                    function timedActionNoCallback(ctx, payload) {
+                        return timedAction(ctx, payload);
```

Comment:
Ah, you're right, I thought you were directly calling the action just w/o a callback.